src/PoshGitTypes.ps1

```diff
@@ -273,13 +273,14 @@ class PoshGitPromptSettings {
     [string]$DescribeStyle = ''
     [psobject]$WindowTitle = {param($GitStatus, [bool]$IsAdmin) "$(if ($IsAdmin) {'Admin: '})$(if ($GitStatus) {"$($GitStatus.RepoName) [$($GitStatus.Branch)]"} else {Get-PromptPath}) ~ PowerShell $($PSVersionTable.PSVersion) $([IntPtr]::Size * 8)-bit ($PID)"}
 
-    [PoshGitTextSpan]$DefaultPromptPrefix       = ''
+    [PoshGitTextSpan]$DefaultPromptPrefix       = '$(Get-PromptConnectionInfo)'
     [PoshGitTextSpan]$DefaultPromptPath         = '$(Get-PromptPath)'
     [PoshGitTextSpan]$DefaultPromptBeforeSuffix = ''
     [PoshGitTextSpan]$DefaultPromptDebug        = [PoshGitTextSpan]::new(' [DBG]:', [ConsoleColor]::Magenta)
     [PoshGitTextSpan]$DefaultPromptSuffix       = '$(">" * ($nestedPromptLevel + 1)) '
 
     [bool]$DefaultPromptAbbreviateHomeDirectory = $true
+    [string]$DefaultPromptConnectionFormat      = '[{1}@{0}]: '
```

Comment:
I wonder if instead of this setting that isn't perhaps obviously connected to `Get-PromptConnectionInfo`, we should just have that command take a format string as parameter.  We could make that parameter optional (default it to `'[{1}@{0}]: ' ` but I'd use it explicitly for `$DefaultPromptPrefix` so folks see they can tweak the format e.g.:
```
[PoshGitTextSpan]$DefaultPromptPrefix       = '$(Get-PromptConnectionInfo -Format "[{1}@{0}]: ")' 
```

Although at this point maybe we switch the format placeholders to `[{01}@{1}]: `?

Reply:
I think `-Format` is better and we should definitely provide a default.  The only reason I would specify the `-Format …` parameter in that setting is purely pedagogical.


Re `DefaultPromptAbbreviateHomeDirectory`, sounds good to me.  We should note that in the CHANGELOG as a potentially breaking change though.

Reply:
Eh, `DefaultPromptAbbreviateHomeDirectory` has been established since v0.7.0, doesn't seem worth breaking everyone now.

I'll replace the new setting with `-Format`.